components/raftstore/src/store/peer_storage.rs

```diff
@@ -2553,7 +2555,7 @@ mod tests {
         // Sync if context is marked sync.
         let context = ProposalContext::SYNC_LOG.to_vec();
         let mut e = Entry::default();
-        e.set_context(context);
+        e.set_context(context.into());
```

Comment:
Perhaps static slice can be used.

Reply:
`&'static [ProposalContext::SYNC_LOG]` is not allowed. To use static slice we need to add
```rust
fn ProposalContxt::sync_log() -> &'static [u8] { ... }
```
 Considering it's only a test, I prefer to keep it.